src/Altinn.Apps/KubernetesWrapper/Controllers/DeploymentsController.cs

```diff
@@ -0,0 +1,51 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using KubernetesWrapper.Services.Interfaces;
+using Microsoft.AspNetCore.Cors;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+
+namespace KubernetesWrapper.Controllers
+{
+    /// <summary>
+    ///  Controller containing all actions related to kubernetes deployments
+    /// </summary>
+    [Route("/[controller]")]
```

Comment:
are we not using the standard api/vi/[controller] as we have discussed before?

Reply:
Fixed.